avm/res/event-grid/namespace/ca-certificate/main.json

```diff
@@ -54,9 +54,7 @@
         "description": "[parameters('description')]",
         "encodedCertificate": "[parameters('encodedCertificate')]"
       },
-      "dependsOn": [
-        "namespace"
-      ]
+      "dependsOn": ["namespace"]
```

Comment:
I'm sorry to say, but it's getting worse 😄 

@fabmas, I recommend to check the `files` view of this PR to see where else changes applied, instead of me commenting them individually.  

To undo those changes, do not use VSCode. It appears your workspace very much wants to reformat the files. Instead, I only see notepad as a reasonable alternative.

@eriqua, I guess we shoud also add some JSON settings to the `.vscode/settings.json`

Reply:
@AlexanderSehr agree, currently the formatting of json and other file types is still relying on prettier, which has multiple references in other files as well. Once prettier is fully removed, we can replace with our rules in the settings.json